src/Microsoft.ML.Data/Depricated/Instances/HeaderSchema.cs

```diff
@@ -379,8 +377,9 @@ public Dense(int count, string[] names)
                 if (size > 0)
                     Array.Copy(names, _names, size);
 
-                _schema = RoleMappedSchema.Create(new FeatureNameCollectionSchema(this),
-                    RoleMappedSchema.CreatePair(RoleMappedSchema.ColumnRole.Feature, RoleMappedSchema.ColumnRole.Feature.Value));
+                // REVIEW: This seems wrong. The default feature column name is "Features" yet the role is named "Feature".
```

Comment:
>// REVIEW: This seems wrong. The default feature column name is "Features" yet the role is named "Feature". [](start = 16, length = 107)

Should we move this review to RoleMappedSchema file?

Reply:
I'm not sure about that. I think it's fine to have the role be "feature", but the column name be "features." What I meant was, this *very* specific usage of it in this file.

---
In reply to: [200457828](https://github.com/dotnet/machinelearning/pull/496#discussion_r200457828) [](ancestors = 200457828)